android_gradle/app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Where do all the icon files come from? Do we need to add them? Should we just create a circle with an A in them so the copyright is amber authors?

Reply:
Those are default icons. It seems to work without, so I've removed all the resource files.